docs/shared/core-features/run-tasks.md

```diff
@@ -102,6 +102,42 @@ npx nx affected -t test
 
 Learn more about the affected command [here](/concepts/affected).
 
+## Run Root-Level Tasks
+
+Sometimes you have tasks that apply to the entire codebase rather than to a single project. But you still want those to go through the "Nx pipeline" to benefit from caching. You can define these in the root-level `package.json` as follows:
```

Comment:
```suggestion
Sometimes you have tasks that apply to the entire codebase rather than to a single project. But you still want those tasks to go through the "Nx pipeline" in order to benefit from caching. You can define these in the root-level `package.json` as follows:
```